content/blog/2020/Weeknotes3.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Week3-20200607"
+date: 2020-06-15T19:08:09-04:00
+categories: ["RIT","Weeknotes","Web Development"]
+---
+As mentioned in the previous week, I learned the basics of sqlite database using the website and simultaneously created the basic todo webapp using the Youtube tutorial.
+### **SQLite learning process**
+The website uses a good strategy **(Spaced Repetition system)** to teach the required content and only 16 lessons are free. So it takes the whole week to progress through the free lessons.
+>See previous weeknote for the site's link
```

Comment:
Don't make it hard for your readers. It doesn't cost you anything to add a link to the site here. 

Also make each weeknote self-contained. Don't make it a continuation of the previous note. 

For instance, don't assume that someone is going to read your weeknotes in order.